app/search_builders/search_builder.rb

```diff
@@ -3,6 +3,6 @@ class SearchBuilder < Hydra::SearchBuilder
 
   def only_images_and_collections(solr_parameters)
```

Comment:
Can you rename this method?  Since there are ETDs here (and no collections), the name is misleading.